settings/defaults.py

```diff
@@ -1,10 +1,22 @@
+# New tokens can be found at https://archive.org/account/s3.php
+
+DOI_FORMAT = '10.70102/fk2osf.io/{guid}'
+
+DATACITE_USERNAME = ''
+DATACITE_PASSWORD = ''
+DATACITE_URL = 'https://doi.test.datacite.org/'
+DATACITE_PREFIX = '10.70102'  # Datacite's test DOI prefix -- update in production
+
 CHUNK_SIZE = 1000
-OSF_API_URI = 'http://localhost:8000/'
+PAGE_SIZE = 100
+OSF_API_URI = 'https://localhost:8000/'
```

Comment:
```suggestion
```

Reply:
Looks like this [isn't being used in the codebase](https://github.com/CenterForOpenScience/osf-pigeon/search?q=OSF_API_URI&unscoped_q=OSF_API_URI) anywhere.